embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/getter/TimestampColumnGetter.java

```diff
@@ -23,7 +23,7 @@ protected void fetch(ResultSet from, int fromIndex) throws SQLException
     {
         java.sql.Timestamp timestamp = from.getTimestamp(fromIndex);
         if (timestamp != null) {
-            value = Timestamp.ofEpochSecond(timestamp.getTime() / 1000, timestamp.getNanos());
+            value = Instant.ofEpochSecond(timestamp.getTime() / 1000, timestamp.getNanos());
```

Comment:
you could use directly `value = timestamp.toInstant()`

Reply:
Good catch!